### PR TITLE
review status indicator

### LINKED ISSUE #1299: Table data / ODP review status indicators
- [x] TableData
- [x] ODP

### PR DESCRIPTION
**Pull request by rfontanarosa**
part of #1299

Schermata 2022-05-10 alle 14 28 35

### CODE REVIEW

File: client/pages/Assessment/Assessment.tsx [hunk 1]
```diff
@@ -40,6 +41,10 @@ const SectionWrapper: React.FC = (props) => {
     )
   }, [countryIso, assessmentName, cycleName, section])
 
+  useEffect(() => {
+    dispatch(ReviewActions.getReviewStatus({ countryIso, assessmentName, cycleName, section }))
```
Comment (minotogna) [inline client/pages/Assessment/Assessment.tsx:45]: fetch only if there's a user logged in

File: common/api/endpoint/ApiEndPoint.ts [hunk 2]
```diff
@@ -33,6 +33,8 @@ export const ApiEndPoint = {
 
     country: () => apiPath('country'),
 
+    review: () => apiPath('review'),
```
Comment (minotogna) [inline common/api/endpoint/ApiEndPoint.ts:36]: add an object  Review and not under Assessment 
review : {
status : { many : ()=> apiPath('review','status')}
}

File: server/api/assessment/getReviewStatus.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+import { CycleDataController } from '@server/controller/cycleData'
+import Requests from '@server/utils/requests'
+
+export const getReviewStatus = async (req: Request, res: Response) => {
+  try {
+    const { countryIso, assessmentName, cycleName, section } = <Record<string, string>>req.query
+
+    const { assessment, cycle } = await AssessmentController.getOneWithCycle({ name: assessmentName, cycleName })
+
+    const returnedOriginalDataPoint = await CycleDataController.getReviewStatus({
```
Comment (minotogna) [inline server/api/assessment/getReviewStatus.ts:15]: returnedOriginalDataPoint might come from copy and paste ;) call it reviewStatus

**Review by minotogna — CHANGES_REQUESTED** [review]
looks good 👍  just some minor issues

File: server/api/assessment/getReviewStatus.ts (hunk 3 above)
Comment (rfontanarosa) [reply server/api/assessment/getReviewStatus.ts:15]: O_O

**Pull request closed without merging**